### PR TITLE
fixes the gripper vertical bar disappearance

### LINKED ISSUE #5791: Details Header ColumnDragDrop CSS issues
DetailsHeader column drag drop issues:
1. Border fade out is not working, which was working before.
2. Gripper ICON on hover of draggable columns is not coming

https://developer.microsoft.com/en-us/fabric#/components/detailslist
In above link, in drag drop example, after enabling column reorder, drag any column, the border will be faded out automatically. But its not happening in the current code. And on hover, gripper ICON is also missing, which was visible before.

Snapshot of production experience:
image

Snapshot of current experience(with issues mentioned above):
image

Requested priority: High

I think the bug has been introduced as part of this PR:
https://github.com/OfficeDev/office-ui-fabric-react/pull/5490

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Fixes #5791. I cannot repro the first symptom of the border not fading away on my side. I've verified that the gripper didn't show up as you hover. I've addressed here in this PR. The subcomponentstyles function cannot handle hover state as a prop - this state comes from CSS itself. This means we had to do some global name styling for the gripper only.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by kenotron** [comment]
I'm taking the approach of not refactoring column to mergestyles / subcomponentstyle yet to fix an urgent bug.

**Comment by kenotron** [comment]
Okay, I've refactored it.

**Comment by kenotron** [comment]
Yay it works! http://fabricweb.z5.web.core.windows.net/pr/refs/pull/5810/merge/#/examples/detailslist/DragDrop

File: packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGenerator.doc.tsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import * as React from 'react';
-import './ThemeGeneratorPage.global.scss';
 import { BaseComponent } from 'office-ui-fabric-react/lib/Utilities';
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGenerator.doc.tsx]: :)

**Pull request merged**